src/mlpack/tests/cf_test.cpp

```diff
@@ -451,8 +451,8 @@ template<>
 void Train<>(RegSVDPolicy& decomposition)
 {
   // Generate random data.
-  arma::mat randomData(100, 100);
-  randomData.randu();
+  arma::mat randomData = arma::zeros(100, 100);
```

Comment:
Haven't thought about it before, but we could also just use ``arma::mat randomData(100, 100, arma::fill::eye);``, it's a little bit shorter.